bowling/tests/argumentParser.ut.cpp

```diff
@@ -0,0 +1,210 @@
+#include "catch.hpp"
+
+#include <string>
+
+#include "argumentParser.hpp"
+
+using Catch::Matchers::Equals;
+
+using ArgNumber = ArgumentParser::ArgumentNumber;
+
+SCENARIO("program arguments and their number should be returned after parsing", "argumentParser")
+{
+    std::string expectedEmpty{};
+
+    SECTION("Test with null arguments list")
+    {
+        int testArgc = 0;
+        const char** testArgv = NULL;
```

Comment:
```suggestion
        const char** testArgv = nullptr;
```
Replace it globally, everywhere